src/system/system.kathleen.make

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
There was a suggestion from @davidbowler to add some info about the system in the header